### PR TITLE
fix: npm run site

### PR DESCRIPTION
**Pull request by hengkx**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: site/theme/template/Layout/Header/Navigation.tsx [hunk 1]
```diff
@@ -68,7 +69,9 @@ export default ({
     );
   }
 
-  const isGitee = document.location.host.indexOf('gitee') !== -1;
+  React.useEffect(() => {
```
Comment (yoyo837) [inline site/theme/template/Layout/Header/Navigation.tsx:72]: 类似 http://xxxgiteexxx.com 这种咋整？

Comment (yoyo837) [reply site/theme/template/Layout/Header/Navigation.tsx:72]: 不过也没这样的域名部署 该站点。

Comment (hengkx) [reply site/theme/template/Layout/Header/Navigation.tsx:72]: 就之前的 改的. 用url?

Comment (hengkx) [reply site/theme/template/Layout/Header/Navigation.tsx:72]:
```
document.location.href.indexOf('gitee') !== -1
```

Comment (hengkx) [reply site/theme/template/Layout/Header/Navigation.tsx:72]: 不对啊 http://xxxgiteexxx.com  这种 也是可以  匹配出的

Comment (yoyo837) [reply site/theme/template/Layout/Header/Navigation.tsx:72]: 我的意思是与antd站点部署无关的，但是刚好碰巧关键词有gitee的，不应该匹配上，但是这种可能性比较小。

**Pull request merged**